vizro-core/src/vizro/models/_action/_action.py

```diff
@@ -59,6 +69,50 @@ def build(self):
             "action_finished": Output("action_finished", "data", allow_duplicate=True),
         }
 
+        action_components = _get_action_callback_mapping(
+            action_id=self.id, argument="components"  # type: ignore[arg-type]
```

Comment:
Can we get rid of the `ignore` here? Do you just need to do `ModelID(self.id)`?

Reply:
Oooh I see, that is a bit ugly. I think `ModelID(str(self.id))` is still better than the ignore though 🙂